cli/src/fmt/simple/operation/mod.rs

```diff
@@ -0,0 +1,12 @@
+use fmt::Render;
+use super::Simple;
+use stellar_client::resources::Operation;
+
+impl Render<Operation> for Simple {
+    fn render(&self, op: &Operation) -> Option<String> {
+        let mut buf = String::new();
+        append_to_buffer!(buf, "ID:   {}", op.id());
+        append_to_buffer!(buf, "Kind: {}", op.kind_name());
```

Comment:
Perhaps we can expand on this with the various details of the operations?

Reply:
I added a separate issue for this as it is going to be a big and obnoxious lift.  Feel free to tackle in this PR or we can come back to it later. https://github.com/kbacha/stellar-sdk/issues/203

Reply:
I'm happy to tackle it. Three of them will be taken care of here: https://github.com/kbacha/stellar-sdk/pull/200